src/storage/event_listener.cc

```diff
@@ -49,6 +59,14 @@ std::string CompressType2String(const rocksdb::CompressionType type) {
   return "unknown";
 }
 
+std::string ExtractSSTFileNameFromError(const std::string &error) {
+  auto match_results = util::RegexMatch(error, "(.*)(/\\w*\\.sst)(.*)");
```

Comment:
IMO regex_search seems better here, and you do not need to capture the left and right part even if you use regex_match. `(.*)(/\\w*\\.sst)(.*)` -> `.*(/\\w*\\.sst).*`

Reply:
Good idea